html/user-profile.html

```diff
@@ -5,35 +5,79 @@
     <meta content="text/html; charset=UTF-8">
     <link rel="stylesheet" type="text/css" href="../css/style.css" />
 </head>
-<body>
-  <header>
-    <nav>
-      <ul>
-        <li><a href="user-profile.html">User Profile</a></li>
-        <li><a href="user-list.html">User List</a></li>
-        <li><a href="create-user.html">Create User</a></li>
-        <li><a href="edit-user.html">Edit User</a></li>
-      </ul>
-    </nav>
-  </header>
-  <table> 
-      <h3>User Profile</h3>
-    <tr>
-      <td>ID:</td>
-      <td>#0000</td>
-    </tr>
-    <tr>
-      <td>Name:</td>
-      <td>John Smith</td>
-    </tr>
-    <tr>
-      <td>Phone:</td>
-      <td>000-000-0000</td>
-    </tr>
-    <tr>
-      <td>Email:</td>
-      <td>john.smith@banno.com</td>
-    </tr>
-  </table>
-</body>
+  <body>
+    <header>
+      <nav>
+        <ul>
+          <li><a href="user-profile.html">User Profile</a></li>
+          <li><a href="user-list.html">User List</a></li>
+          <li><a href="create-user.html">Create User</a></li>
+          <li><a href="edit-user.html">Edit User</a></li>
+        </ul>
+      </nav>
+    </header>
+    <h3>User Profile</h3>
+    <table> 
+      <tr>
+        <td>ID:</td>
+        <td>#0000</td>
+      </tr>
+      <tr>
+        <td>Name:</td>
+        <td>John Smith</td>
+      </tr>
+      <tr class="expand">
+        <td>Phone:</td>
+        <td>000-576-5309</td>
+      </tr>
+      <tr class="expand">
+        <td>Email:</td>
+        <td>john.smith@banno.com</td>
+      </tr>
+      <tr>
+        <td>
+          <button onclick="cardExpandCollapse(this)">Collapse</button>
+        </td>
+      </tr>
+    </table>
+    <table> 
+      <tr>
+        <td>ID:</td>
+        <td>#0000</td>
+      </tr>
+      <tr>
+        <td>Name:</td>
+        <td>John Smith</td>
+      </tr>
+      <tr class="expand">
+        <td>Phone:</td>
+        <td>000-576-5309</td>
+      </tr>
+      <tr class="expand">
+        <td>Email:</td>
+        <td>john.smith@banno.com</td>
+      </tr>
+      <tr>
+        <td>
+          <button onclick="cardExpandCollapse(this)">Collapse</button>
+        </td>
+      </tr>
+    </table>
+    <script>
+      function cardExpandCollapse(element) {
+        const button = element;
+        const collapses = element.parentNode.parentNode.parentNode.querySelectorAll(".expand");
+
+        collapses.forEach (collapse => {
+          collapse.classList.toggle('collapse');
```

Comment:
I hadn't messed with the `classList` property, so new learning for me today!

This adds or removes the collapse class, but the expand class is still in place. Could you also toggle the expand class here?

Reply:
You could also give the elements an id of collapsable and target that with your query selector on line 69 and then toggle the collapse class. I just don't like the idea of having expanded and collapsed as classes at the same time.

Reply:
the problem with the id is that it only allowed me to grab one element at a time rather than hiding both row. and is there a reason toggling the expand class would be better?

Reply:
@phillnab I just don't like the idea of having an expanded and collapsed class at the same time. 

If I were stumbling upon this I would think that the expanded state represents something when it is expanded. As you currently have it it could still have the expand class when it's actually collapsed.

Even setting the class to something like collapsable-row is better imo because then someone coming along later to make changes won't confuse that with styles affecting a state.